### PR TITLE
Update README with new Clean Code wording and link from Marketing

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ SonarLint IntelliJ Plugin
...
 
 Useful links
 ------------
 
 - [Documentation](https://docs.sonarsource.com/sonarlint/intellij/)
-  - A full list of supported programming languages and links to the static code analysis rules associated with each language is available on the [Rules page](https://docs.sonarsource.com/sonarlint/intellij/using-sonarlint/rules/).
+  - A full list of supported programming languages and links to the static code analysis rules associated with each language are available on the [Rules page](https://docs.sonarsource.com/sonarlint/intellij/using-sonarlint/rules/).
 - [Community](https://community.sonarsource.com/c/help/sl)
-  - Report an issue, ask for some help or suggest new features
+  - Report an issue, ask for some help, or suggest new features
```
Comment (kevin-hinz) [inline README.md:16]: The sentence should have a `.` at the end.
Otherwise, the changes LGTM!

**Review by kevin-hinz — APPROVED** [review]
Aside from the missing `.` I commented on, it LGTM!

File: README.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ SonarLint IntelliJ Plugin
 [![Build Status](https://api.cirrus-ci.com/github/SonarSource/sonarlint-intellij.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/sonarlint-intellij)
 [![Quality Gate](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij&metric=alert_status)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij)
 
-SonarLint is an IDE extension that helps you detect and fix quality issues as you write code.
+SonarLint is an IDE extension that helps you detect and fix quality issues as you write [Clean Code](https://www.sonarsource.com/solutions/clean-code/).
```
Comment (damien-urruty-sonarsource) [inline README.md:7]: I have 2 comments on this line:

> helps you detect and fix quality issues as you write Clean Code

I find the phrasing a bit imprecise, I understand it as if people are already writing clean code, not that we help them achieve that. Maybe something like: 

> SonarLint is an IDE extension that supports you in writing Clean Code, by detecting and helping you fix quality issues as you type

My second point is that READMEs are usually read by technical people, so I'm a bit reluctant to point to a marketing website from here, so I would prefer to point to https://docs.sonarsource.com/sonarlint/intellij/concepts/clean-code/

**Review by damien-urruty-sonarsource — COMMENTED** [review]
Thanks for contributing this David. I just left one comment. Out of curiosity, what drove you to contribute to our repo?

File: README.md (hunk 1 above)
Comment (david-cho-lerat-sonarsource) [reply README.md]: Thanks, @kevin-hinz , I've added the period.

**Pull request merged**